tests/glusterGeoRepStatusOld.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Please add glusterGeoRepStatusOld.xml.license file (similar to the *.license files already present) to make the linter happy.

Reply:
Done.